packages/react/src/components/Tag/Tag.js

```diff
@@ -41,19 +41,21 @@ const Tag = ({
     [`${prefix}--tag--filter`]: filter,
   });
   return filter ? (
-    <span
-      role="button"
+    <button
       className={tagClasses}
-      title={title || 'Clear filter'}
-      tabIndex="0" // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
+      aria-label={
+        title !== undefined
+          ? `${title} ${children}`
+          : `Clear filter ${children}`
```

Comment:
Is this so that we don't have a mismatch with the visual and accessible labels? Would there a preferred way to separate these or is this the ideal? Would we ever want something like:

```html
<span class="tag">
  <button>Clear</button>
  Text for tag
</span>
```

Reply:
Well, since we don't _technically_ have a visual label for the X icon remove button this was just so it spoke clearly (as defined in the issue) in VO etc

Reply:
title is -- not what we want here, but removing or not incorporating the prop is a breaking change.